widgetssdk/src/main/res/values/new_strings.xml

```diff
@@ -170,6 +170,10 @@
     <string name="secure_messaging_chat_banner_bottom">@string/message_center_bottom_banner</string>
     <string name="secure_messaging_chat_unavailable_message">@string/message_center_error_unavailable_message</string>
     <string name="secure_messaging_chat_banner_top">@string/need_live_support_label</string>
+    <string name="secure_messaging_chat_leave_current_conversation_title">@string/message_center_leaving_conversation_title</string>
```

Comment:
Do we use references for the new strings?
As I remember, we need to put there just a value.

Reply:
I have no idea what is the right approach.
Is there some discussion that has been done regarding this before?

Reply:
I know only the [String resources key namingagreements](https://glia.atlassian.net/wiki/spaces/ENG/pages/3433726090/String+resources+key+naming+agreements) doc. However, it doesn't say anything about this case. Looks like something that needs clarification, indeed.

Reply:
I will close this PR for now as is. But it would be great to bring this to discussion on tomorrow's standup